nasa/earthdata-search
# EDSC-3417-3418: added duplicate collection notice

# Overview
Let's user know if there's a duplicate collection available

### What is the feature?
Add additional text to data quality summary dropdown in project page letting a user know that a duplicate collection is available.

### What is the Solution?
- Update GraphQL queries to get duplicateCollections
- If there is a duplicate collection, add another summary to the dataQualitySummaries array in ProjectPanels that is then passed to the DataQualitySummary notice/dropdown.

### What areas of the application does this impact?

Project Page

# Testing

### Reproduction steps

- **Environment for testing:**
- **Collection to test with:**

1. Go to /search
2. Search for C1972954180-PODAAC 
3. Add it to your collection (click Green Plus Sign)
4. Go to project page /project
5. Look for "Important data availability information" dropdown and click it
6. See that text appears
7. Click link in text to be transported to search page with collection selected

### Attachments
<img width="1016" alt="Screen Shot 2022-06-16 at 6 04 46 PM" src="https://user-images.githubusercontent.com/4313463/174183689-2025b8ee-34e2-4d02-be85-528744351854.png">


# Checklist
